docs/index.asciidoc

```diff
@@ -58,6 +58,14 @@ input {
 
 |=======================================================================================================================
 
+.Technology Preview
+****
+This {esf-name} input plugin is part of a _Technology Preview_, which means that both configuration options and implementation details are subject to change in minor releases without being preceded by deprecation warnings.
+
+Before upgrading this plugin or Logstash itself, please pay special attention to the changelogs to avoid being caught by surprise.
+****
+
+
 [id="plugins-{type}s-{plugin}-enrichment"]
 ==== Enrichment
 
```

Comment:
I think we could keep also this comment below about @timestamp as internal. ESF sets this field internally and customers do not really have access to it, as of today.